cloudformation/ad/ad-integration.yaml

```diff
@@ -561,7 +565,9 @@ Resources:
                   response_data['Message'] = 'Resource creation successful!'
                   physical_resource_id = create_physical_resource_id()
                   ds.reset_user_password(DirectoryId=directory_id, UserName='ReadOnlyUser', NewPassword=read_only_password)
-                  ds.reset_user_password(DirectoryId=directory_id, UserName=user_name, NewPassword=user_password)
+                  for name in user_names.split(","):
+                    name = name.strip()
```

Comment:
It's ok to strip here, but just for sharing: in this way you strip once per every username. If you strip `user_names` when defining the loop instead you strip just once for all.